Source/rotation/rotation_sources.cpp

```diff
@@ -178,53 +182,6 @@ Castro::corrrsrc(const Box& bx,
       dx[i] = 0.0_rt;
   }
 
-  auto omega = get_omega();
-
-  Real dt_omega[3];
-
-  Array2D<Real, 0, 2, 0, 2> dt_omega_matrix = {};
-
-  if (implicit_rotation_update == 1) {
```

Comment:
what happened to this check on `implicit_rotation_update` ?

Reply:
I think it is missing in fill_dt_omega_matrix

Reply:
`fill_dt_omega_matrix` is called within `if (implicit_rotation_update == 1)` I think.

Reply:
yes,  you are right.